frontend/lib/norent/letter-builder/steps.tsx

```diff
@@ -34,6 +36,17 @@ function isUserOutsideNYC(s: AllSessionInfo): boolean {
   return !isUserInNYC(s);
 }
 
+function isUserInLA(s: AllSessionInfo): boolean {
+  const norent = s.norentScaffolding;
+  if (!(norent && norent.zipCode)) return false;
+  if (isUserInNYC(s)) return false;
+  return zipCodeIsInLosAngeles(norent.zipCode);
+}
+
+function isUserOutsideLA(s: AllSessionInfo): boolean {
+  return !isUserInLA(s);
+}
+
```

Comment:
writing two functions here may be a bit redundant, as we may never have a use for the `isUserInLA` function, but it felt like the clearest way to spell out the logic.

Reply:
Yeah, I agree that doing this kind of thing for our "session predicate" functions makes things more readable!